mantidproject/mantid
# Reformat and add cross-references

Add links to various items in the `rst` files for `CreateCacheFilename` and `ClearCache`.

**To test:**

Build the docs and see that there are links.

*There is no associated issue.*

*Does not need to be in the release notes* since it is a minor documentation fix.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
